pepper-apis/docs/specification/src/components/responses.yml

```diff
@@ -75,12 +87,30 @@ ActivityAnswersPatchResponse:
             type: array
             items:
               $ref: '../pepper.yml#/components/schemas/ValidationFailure'
-ActivityAnswersPatchErrorResponse:
-  description: activity instance is read-only or answer submission failed validation
+ActivityAnswersPatchBadPayloadResponse:
+  description: answer submissions payload is invalid
   content:
     application/json:
       schema:
-        $ref: '../pepper.yml#/components/schemas/Error.Answer'
+        $ref: '../pepper.yml#/components/schemas/Error.ActivityAnswersPatchBadPayload'
+ActivityAnswersPatchExistsResponse:
+  description: answer already exists for question, `answerGuid` should be specified
```

Comment:
Am I correct in assuming this means we are enforcing 1 answer per question?

Reply:
Yeah, that has always been the assumption.